stdlib/pervasives.gr

```diff
@@ -1,19 +1,26 @@
+/* grainc-flags --no-pervasives */
```

Comment:
Why did you choose to do this inside the file instead of skip adding it inside the compiler?

Reply:
We'd have to detect that we were compiling Pervasives, and that felt a little brittle to me (what happens if you import Pervasives manually? what happens if you name another module `Pervasives`?), but this was a surefire way to make sure the flag was applied specifically to this file.

Reply:
I think that's fine. If we ever allow (or do we now?) people to swap out pervasives, it seems weird that they'd need to know to add this flag to the top.

Reply:
That's going to be a fairly involved change, so I made an issue: https://github.com/grain-lang/grain/issues/1632